.idea/poetry.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should this file be committed?

Reply:
Is this from the Poetry plugin?

Reply:
I don't think this should be committed. My Poetry isn't even at that path